### PR TITLE
Covid Statistic

### PR DESCRIPTION
**Pull request by ahmetbozaci**
- [README](https://github.com/ahmetbozaci/react-capstone/blob/develop/README.md)
- [LIVE DEMO](https://covid-world-statistic.netlify.app/)
- [VIDEO](https://www.loom.com/share/bf3a5f6824314c988f03f0ad1b4ffef1)

**In this project**

- I create two pages: `home `and `detail` 
   - The home page includes the number of new covid cases of 20 random countries. And every country is clickable so when ı click on the specific country details page opening.
   - Details page includes a specific country's covid statistic.
   - Details page has a back button top-left of the page. It opens the home page with new 20 countries.
 - I use functional components when creating components.
 - I use duck pattern to create redux action, type, reducer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+https://creativecommons.org/licenses/by-nc/4.0/
```
Comment (bimbolabuari) [inline README.md]: As required, let's give appropriate credit to the author [Nelson Sakwa on Behance.](https://www.behance.net/sakwadesignstudio) 👍

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <Col xs={6} className="p-3 fw-bolder fs-3">
+          <p className="mt-5">
+            WORLD
+            <br />
+            TOTAL CASE
+          </p>
+          {worldData.map((item) => (
+            <p key={uuidv4()} className="fw-normal mb-5">{item['Total Cases_text']}</p>
+          ))}
+          {/* <p className="fw-normal mb-5">{state[0][0]['Total Cases_text']}</p> */}
```
Comment (bimbolabuari) [inline src/components/Home.js]: Let's remove commented code as it makes the code look unclean and unprofessional 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @ahmetbozaci, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Descriptive Pull Request 👍 
-  Deployed Application 💯 
- Nice video Presentation ✔️ 
- You followed the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) required 🚀 
- Interactions
  - Home page 🎉 
     - When the page loads, the web app shows the list of categories that could be filtered out by some parameter.
     - Along with the category name, you displayed some numeric values per category that come from the API.
     - When the user clicks on a category item, the application navigates to the details page.
   - Details page 🎉 
     - On the details page, the web app retrieves data from the API to show detailed data in the category.
    - When the user clicks on the "Back" button (<), the user navigates to the home page.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**